bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableClusterUtilities.java

```diff
@@ -73,6 +70,66 @@ public static BigtableClusterUtilities forAllInstances(String projectId)
   }
 
   /**
+   * @return The instance id associated with the given project, zone and cluster.
+   */
```

Comment:
Can we add a comment here saying that we expect instance and cluster to have one-to-one relationship? I know this for HBase right now, but it doesn't hurt to clarify.